virttest/qemu_vm.py

```diff
@@ -899,9 +899,9 @@ def add_memorys(devices, params):
             options, devs = [], []
             params.setdefault("mem", "512")
             mem_params = params.object_params("mem")
-            mem_params.setdefault("automem", "yes")
+            mem_params.setdefault("automem", "no")
             automem = mem_params["automem"] == "yes"
-            if automem:
+            if automem and mem_params.get("max_usable_mem"):
```

Comment:
OK, this prevents the bug, but I'm wondering why are you not evaluating the `max_usable_mem` here, rather than during the `env_process`?